records_mover/records/mover.py

```diff
@@ -126,6 +127,8 @@ def move(records_source: RecordsSource,
     elif (isinstance(records_source, SupportsMoveToRecordsDirectory) and
           isinstance(records_target, MightSupportMoveFromTempLocAfterFillingIt) and
           records_source.has_compatible_format(records_target) and
+          records_source.
+            can_move_to_scheme(records_target.temporary_loadable_directory_scheme()) and
```

Comment:
These three lines added lines are the meat of this PR.  They result in us not trying to do bulk exports when we'd need a temporary bucket and don't have one.  As a result, we move on and try a less efficient approach rather than dying with an exception.